test/junit/scala/concurrent/duration/DurationTest.scala

```diff
@@ -0,0 +1,27 @@
+package scala.concurrent.duration
```

Comment:
IMO this test should instead of being its own file, be added to [duration-tck](https://github.com/scala/scala/blob/2.12.x/test/files/jvm/duration-tck.scala)